packages/EasyCodingStandard/packages/ChangedFilesDetector/src/Cache/CacheFactory.php

```diff
@@ -8,6 +8,16 @@
 
 final class CacheFactory
 {
+    /**
+     * @var string
+     */
+    private $cacheDirectory;
+
+    public function __construct(string $cacheDirectory)
+    {
+        $this->cacheDirectory = $cacheDirectory;
```

Comment:
maybe we can check if it is writable?

Reply:
It's checked when used by: `FileSystem::createDir($cacheDirectory);` in `create()` method.
This method reports any occured error. 

What would be advantange of doing it on 2 places?